include/devkitpro_defs.h

```diff
@@ -0,0 +1,12 @@
+#ifndef DEVKITPRO_DEFS__
```

Comment:
Please do not use fiercely protected trademarks in BlocksDS PRs.

Can we just make building maxmod for NDS, well, call libnds?

Reply:
Yeah, we probably should. The ASM just has "random" references to it without any kind of includes, which is very annoying.

Reply:
This is valid and expected - you cannot use C includes inside ASM code; you just reference symbols directly. These symbols should be included via proper headers when writing C code.